dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/LDAPRegressionTest.java

```diff
@@ -375,4 +376,26 @@ public void testPrincipalSearchExpressionWithParen() throws Exception {
         assertEquals(1, listResults.size());
         assertEquals("uid=Bob (Contractor),o=ibm,c=us", listResults.get(0));
     }
+
+    /**
+     * Temporary fat test to ensure our Beta fence for bindAuthmechanism/GSSPI/Kerberos is working as expected.
+     *
+     * Can be removed when betaFence code is removed from LdapConnection
+     *
+     * Also remove CWWKE0701E from teardownClass
+     */
+    @Test
+    public void testBetaFenceForBindAuthMech() throws Exception {
```

Comment:
This is fine here for now, but I think we will want this in a KRB5 specific test when you get a chance. In fact, I would create a whole new `com.ibm.ws.security.wim.adapter.ldap_fat.krb5` project that contains the KRB5 fats.

Reply:
Yes, that was my thought as well -- create a new Kerberos bucket. I will remove this test when we go to GA.